apps/crn-frontend/src/__fixtures__/algolia.ts

```diff
@@ -21,11 +22,14 @@ export const createAlgoliaResponse = <EntityType extends keyof EntityResponses>(
   renderingContent: {},
   processingTimeMS: 1,
   ...overrides,
-  hits: data.map((item, i) => ({
-    ...item,
-    objectID: `${i}`,
-    __meta: { type },
-  })),
+  hits: data.map(
+    (item, i) =>
```

Comment:
> (item, i): EntityHit<EntityType> =>

Reply:
yeah that would only work if the input types were dont differently - ie the `data` and `type` were actually tied together. Technically at the moment you can give it a type that does not correspond to the data which is why typescript moans. Like I said its just a fixture so I wouldn't get too hung up on it and simply leave the cast

Reply:
That's not going to work

Reply:
@kmaid If you have better solution on your mind please push it into this branch/PR :)